crates/runtime/src/env/mod.rs

```diff
@@ -73,6 +70,8 @@ where
 {
     accounts: T::AccountStore,
     templates: T::TemplateStore,
+    /// A [`PriceResolverRegistry`] for calculating gas prices.
```

Comment:
do we want this comment? (it's a private field)

Reply:
Ups, left it since I initially made it public. Removing it